install_plugin_windows.bat

```diff
@@ -0,0 +1,46 @@
+:: Copyright © 2024 Seagate Technology LLC and/or its Affiliates
+@echo off
+
+:: Find the installer file
+for %%I in (cloudfuse*.exe) do (
+    set "installer=%%I"
+    break
+)
+
+:: Check if the installer file exists and if so install it
+if exist "%installer%" (
+    echo Installing Cloudfuse
+    "%installer%" /VERYSILENT /SUPPRESSMSGBOXES /NORESTART
+) else (
+    echo Installer file not found
+    exit /b
+)
+
+echo Installing plugin
+
+:: Stop the VMS server
+set serviceName="metavmsMediaServer"
+echo Attempting to stop %serviceName%
+sc stop %serviceName% >NUL 2>&1
+echo Service stopped successfully.
+
+:: Copy the plugin file
+echo Attempting to copy the plugin file
+copy ".\cloudfuse_plugin.dll" "C:\Program Files\Network Optix\Nx Meta\MediaServer\plugins\"
+if %errorlevel% neq 0 (
+    echo Failed to copy the plugin file
+    exit /b
+)
+
+:: Restart the VMS
+echo Attempting to start %serviceName%
+sc start %serviceName% >NUL 2>&1
+if %errorlevel% neq 0 (
+    echo Failed to start %serviceName%
+    exit /b
+)
```

Comment:
This sequence:

1. stops the service, then
2. copies the plugin, and
3. starts the service

I think it would be more reliable to:

1. copy the plugin
2. restart the service (or stop and start it)

That way, a failed copy won't leave the VMS stopped.

Reply:
Windows doesn't have a restart mechanism, just start and stop. I did change the process so that it should try to restart the VMS if the plugin copy failed.